herokuappTests/utility/booking_api_utils.py

```diff
@@ -0,0 +1,16 @@
+from request_utils import RequestUtils
+
+class BookingApiUtils:
+    def __init__(self, token=None):
+        self.token = token
+        self.request = RequestUtils()
+
+    def auth_headers(self):
+        return {
+            "Content-Type": "application/json",
+            "Cookie": f"token={self.token}"
+        }
+
+    def create_booking(self, **kwargs):
+
+        return self.request.post("/booking", headers=self.auth_headers(), json=kwargs)
```

Comment:
It is better to formulate payload content here so that you cannot pass any kind of dict here.
On the other hand, you can use kwargs to resolve attributes from the upstream to the structure that you will send here.

Example:
payload = {
  "firstname": kwargs.get("firstname", "")
  "lastname": kwargs.get("lastname", "")
   ...
}